From: Michael Olbrich <m.olbrich@pengutronix.de>
Date: Fri, 8 Oct 2010 20:13:40 +0200
Subject: [PATCH] add missing subdirs

SUBDIRS is not set correctly in several places in examples
This patch fixes it.

Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>
---
 examples/declarative/modelviews/modelviews.pro |    1 +
 examples/examples.pro                          |    2 +-
 examples/itemviews/itemviews.pro               |    4 +++-
 examples/tutorials/tutorials.pro               |    1 +
 4 files changed, 6 insertions(+), 2 deletions(-)

diff --git a/examples/declarative/modelviews/modelviews.pro b/examples/declarative/modelviews/modelviews.pro
index 00e07fe..7f7d673 100644
--- a/examples/declarative/modelviews/modelviews.pro
+++ b/examples/declarative/modelviews/modelviews.pro
@@ -1,6 +1,7 @@
 TEMPLATE = subdirs
 
 SUBDIRS += \
+    abstractitemmodel \
     objectlistmodel \
     stringlistmodel \
     abstractitemmodel \
diff --git a/examples/examples.pro b/examples/examples.pro
index 67b4cf8..e67030a 100644
--- a/examples/examples.pro
+++ b/examples/examples.pro
@@ -72,7 +72,7 @@ contains(QT_CONFIG, declarative): SUBDIRS += declarative helper
 win32:!win32-g++*: SUBDIRS += activeqt
 contains(QT_CONFIG, xmlpatterns):!contains(QT_CONFIG, no-gui): SUBDIRS += xmlpatterns
 contains(DEFINES, QT_NO_CURSOR): SUBDIRS -= mainwindows
-contains(QT_CONFIG, concurrent): SUBDIRS += qtconcurrent
+!contains(DEFINES, QT_NO_CONCURRENT): SUBDIRS += qtconcurrent
 
 # install
 sources.files = README *.pro
diff --git a/examples/itemviews/itemviews.pro b/examples/itemviews/itemviews.pro
index 137599c..547694d 100644
--- a/examples/itemviews/itemviews.pro
+++ b/examples/itemviews/itemviews.pro
@@ -2,6 +2,7 @@ TEMPLATE      = subdirs
 SUBDIRS       = addressbook \
                 basicsortfiltermodel \
                 chart \
+                coloreditorfactory \
                 combowidgetmapper \
                 customsortfiltermodel \
                 dirview \
@@ -13,7 +14,8 @@ SUBDIRS       = addressbook \
                 simpledommodel \
                 simpletreemodel \
                 simplewidgetmapper \
-                spinboxdelegate
+                spinboxdelegate \
+                stardelegate
 
 # install
 sources.files = README *.pro
diff --git a/examples/tutorials/tutorials.pro b/examples/tutorials/tutorials.pro
index ba1769d..cb1ca6e 100644
--- a/examples/tutorials/tutorials.pro
+++ b/examples/tutorials/tutorials.pro
@@ -1,5 +1,6 @@
 TEMPLATE      = subdirs
 SUBDIRS       = \
+                widgets \
                 addressbook \
                 modelview \
                 threads
